packages/gateway/src/features/registration/schema.graphql

```diff
@@ -552,6 +554,7 @@ type Query {
     locationId: String
     event: String
   ): RegistrationCountResult
+  fetchRecordsDetailsByCompositionId(id: ID): RecordDetails
```

Comment:
Would something like `fetchRecordDetailsForVerification(compositionId: ID)` be more descriptive? Or `fetchRecordDetailsAnonymously(compositionId: ID)`. What do you think?